### DISCUSSION TITLE
Help with HyperSD config

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#3081] [Q&A] [answered]
I noticed the new version of SDXL model for Fooocus and tried myself.
I easily added the cdebase for HyperSD installation. This is the HyperSD config.

            print('Enter Hyper-SD mode.')
            progressbar(async_task, 1, 'Downloading Hyper-SD components ...')
            loras += [(config.downloading_sdxl_hyper_sd_lora(), 0.8)]

            if refiner_model_name != 'None':
                print(f'Refiner disabled in Hyper-SD mode.')
            refiner_model_name = 'None'
            sampler_name = advanced_parameters.sampler_name = 'dpmpp_sde_gpu'
            scheduler_name = advanced_parameters.scheduler_name = 'karras'
            patch.sharpness = sharpness = 0.0
            cfg_scale = guidance_scale = 1.0
            patch.adaptive_cfg = advanced_parameters.adaptive_cfg = 1.0
            refiner_switch = 1.0
            patch.positive_adm_scale = advanced_parameters.adm_scaler_positive = 1.0
            patch.negative_adm_scale = advanced_parameters.adm_scaler_negative = 1.0
            patch.adm_scaler_end = advanced_parameters.adm_scaler_end = 0.0
            

But the result is bad.
            
![image](https://github.com/lllyasviel/Fooocus/assets/91205658/44f8e891-b363-4024-ab64-42c4d4f40edb)

I think I missed some points for this implementation.
Please let me konw what I missed or misunderstood. Thank you.

**Comment by mashb1t** [comment, marked as answer]
The advanced_parameters implementation has been removed over 3 months now, please refer to the official implementation.

https://github.com/lllyasviel/Fooocus/blob/dfff9b7dcfe447d3595d87c640af78ccf7389eae/modules/async_worker.py#L281-L298
